doc/test-framework/scorecard.md

```diff
@@ -324,8 +325,24 @@ Example of a valid JSON output:
 
 **NOTE:** The `ScorecardOutput.Log` field is only intended to be used to log the scorecard's output and the scorecard will ignore that field if a plugin provides it.
 
+## Running the scorecard with a deployed CSV
+
+The scorecard can be run using only a [Cluster Service Version (CSV)][olm-csv], providing a way to test cluster-ready and non-SDK operators.
+
+Running with a CSV alone requires both the `--csv-path=<CSV manifest path>` and `--olm-deployed` flags are set. The scorecard assumes your CSV and relevant CRD's have been deployed onto the cluster using the OLM when using `--olm-deployed`. This [document][olm-deploy-operator] walks through bundling your CSV and CRD's, deploying the OLM on minikube or [OKD][okd], and deploying your operator. Once these steps have been completed, run the scorecard with both `--csv-path=<CSV manifest path>` and `--olm-deployed` flags.
```

Comment:
Would not minishift be the open shift equivalent of minikube?

Reply:
OKD encompasses both minishift and OpenShift. minishift only supports OpenShift 3.x as of now anyways.

Reply:
No this is fine. 👍